apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -86,9 +106,12 @@ export default class AnimationPickerBody extends React.Component {
       scrollWindow.scrollTop + MAX_HEIGHT >= scrollWindow.scrollHeight * 0.9 &&
       (!pageCount || nextPage <= pageCount)
     ) {
-      const {results: newResults, pageCount} = this.searchAssetsWrapper(
-        nextPage
-      );
+      let {results: newResults, pageCount} = this.searchAssetsWrapper(nextPage);
+      if (this.props.hideBackgrounds) {
+        newResults = newResults.filter(
+          animation => !animation.categories.includes('backgrounds')
+        );
```

Comment:
nit: we filter backgrounds in multiple places now, so we should pull this out into a helper method

also, question: this will currently filter a result that has `["backgrounds", "environments"]` as its categories -- is that what we want? or should we only filter the result if its _only_ category is "backgrounds"?

Reply:
I think that we should filter it out if it has backgrounds as a category because we don't want them to be able to use a background as a sprite right?